obspy/core/tests/test_event.py

```diff
@@ -253,6 +256,38 @@ def setUp(self):
         # Also clear the tracker.
         ResourceIdentifier._ResourceIdentifier__resource_id_tracker.clear()
 
+    def test_read_invalid_filename(self):
+        """
+        Tests that we get a sane error message when calling read_events()
+        with a filename that doesn't exist
+        """
+        doesnt_exist = 'dsfhjkfs'
+        for i in range(10):
+            if os.path.exists(doesnt_exist):
+                doesnt_exist += doesnt_exist
+                continue
+            break
+        else:
+            self.fail('unable to get invalid file path')
```

Comment:
Should say: "Reading this message means you should probably stop calling your files `dsfhjkfsdsfhjkfsdsfhjkfsdsfhjkfsdsfhjkfsdsfhjkfsdsfhjkfsdsfhjkfsdsfhjkfsdsfhjkfs`" 😃